packages/harmony/src/components/button/index.ts

```diff
@@ -0,0 +1,10 @@
+export { Button } from './Button'
+export { PlainButton } from './PlainButton'
+export {
+  ButtonProps,
+  ButtonType,
+  ButtonSize,
+  PlainButtonProps,
+  PlainButtonSize,
+  PlainButtonType
+} from './types'
```

Comment:
should export from src/components/index.ts and src/index.ts. Can also do that in the PR where the web instances are replaced, either way

Reply:
oh good call. I'll do it then